2020-01-Winter/1_linked_lists/README.md

```diff
@@ -67,21 +67,22 @@ Go to [Solution](#s2)   [Top](#top)
 <!-- Don't remove -->
 <a name="p3"/>
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Linked List Duplicates
 
-Source: TODO :bug:
+Source: Cracking The Code Interview Book, Chapter 2, Question 2.1
 
 #### Scenario
 
-Problem Statement TODO :bug:
-
-#### Example Input
+Remove duplicated from an **unsorted** linked list of integers.
+*Follow Up Question:* How would you solve this without a temporary buffer?
 
-If the problem is simple enough, remove this section. TODO :bug:
+**Assume the Tail will NOT connect back to the Head. Meaning it is NOT circular.**
 
 #### Function Signature
 
-TODO :bug:
+`RemoveDuplicatesWithBuffer(Node *Start);`
+
+`RemoveDuplicatesWithoutBuffer(Node *Start)`
```

Comment:
Missing return type here. Also, doing a code block lets you specify which language so you get great syntax highlighting justice.

````
```c++
void RemoveDuplicatesWithBuffer(Node *Start);
void RemoveDuplicatesWithoutBuffer(Node *Start);
```
````